vector/src/main/java/im/vector/activity/VectorWebViewActivity.kt

```diff
@@ -35,35 +42,37 @@ class SimpleWebViewActivity : RiotAppCompatActivity() {
     @BindView(R.id.simple_webview)
     lateinit var webView: WebView
 
-    /* ==========================================================================================
+
+    /* =====================================================================@=====================
      * Life cycle
      * ========================================================================================== */
 
     override fun getOtherThemes() = Pair(R.style.AppTheme_NoActionBar_Dark, R.style.AppTheme_NoActionBar_Black)
 
-    override fun getLayoutRes() = R.layout.activity_simple_web_view
+    override fun getLayoutRes() = R.layout.activity_vector_web_view
 
     override fun initUiAndData() {
         configureToolbar()
+        waitingView = findViewById(R.id.simple_webview_loader)
 
-        webView.settings.let {
+        webView.settings.apply {
```

Comment:
Ok to change from `let` to `apply`, but I had bad experiment when calling a method which exists both in the object and in the current class. There is no call to any method in the apply block, so ok for this change.